tgui/packages/tgui/interfaces/PreferencesMenu/data.ts

```diff
@@ -124,6 +124,9 @@ export type PreferencesMenuData = {
   character_preview_view: string;
   character_profiles: (string | null)[];
 
+  server_export_enabled: boolean;
+  personal_preference_export_available: boolean;
```

Comment:
Not that it matters since I don't want this in the UI yet, but data types should always use BooleanLike instead, because they're sent as 0/1.